Docs/Lightning.md

```diff
@@ -19,64 +19,15 @@ First you need to install c-lightning, to do that follow [this](https://github.c
 
 If you have c-lightning running already you can stop it. `cli/lightning-cli stop`
 
-### Install Tor
+### Install and configure Tor
+Pick the guide below for the device your're running your node on:
+ - [Mac Osx](./Tor.md/#Connecting-over-MacOS)
+ - [Windows 10](./Tor.md/#Connecting-over-Tor-Windows-10)
+ - [Debian 10](./Tor.md/#Connecting-over-Tor-Linux-Debian-10)
 
-You will need Tor running on the node. On linux `sudo apt install tor` works, on macOS `brew install tor` does the trick.
+Find the suggested Authentication settings on the device **running FN** [here](./Authentication.md/#On-the-device-running-FN).
+Find the suggested `bitcoin.conf` settings for FN [here](./Howto.md/#Bitcoin-Core-settings). For *Lightning* see below, or click [here](#Connect-Fully-Noded).
 
-Boot tor as a service:
```

Comment:
why delete this? these are specific to setting up lightning as a hidden service

Reply:
I moved them to the connect-node.md, for the top of my head. I do not delete anything that has purpose, at least not intentionally